packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -131,6 +186,18 @@ const openNewAsset = (assetType: AssetType) => {
 	}
 };
 
+// This conversion should maybe be done in the document component - tera-preview-panel.vue does this conversion differently though...
+const getXDDuri = (docAssetId: string): string =>
```

Comment:
Nitpick: please add a FIXME or TODO and more affirmative observations if you think we should do something here, otherwise the comment is a bit frivolous as it isn't clear what follow up actions should be.